kaspad/src/daemon.rs

```diff
@@ -348,7 +348,7 @@ do you confirm? (answer y/n or pass --yes to the Kaspad command line to confirm
     let outbound_target = if connect_peers.is_empty() { args.outbound_target } else { 0 };
     let dns_seeders = if connect_peers.is_empty() && !args.disable_dns_seeding { config.dns_seeders } else { &[] };
 
-    let grpc_server_addr = args.rpclisten.unwrap_or(ContextualNetAddress::unspecified()).normalize(config.default_rpc_port());
+    let grpc_server_addr = args.rpclisten.unwrap_or(ContextualNetAddress::loopback()).normalize(config.default_rpc_port());
```

Comment:
Did you sanity test local access works fine?

Reply:
Yes:

```
kaspa-utils ./kaspactl -s=127.0.0.1 -a GetInfo     
{
    "getInfoResponse":  {
        "p2pId":  "0a1d9764-baff-4267-b381-ca241d128963",
        "mempoolSize":  "0",
        "serverVersion":  "0.13.3",
        "isUtxoIndexed":  true,
        "isSynced":  true,
        "error":  null
    }
}
```